lerna.json

```diff
@@ -1,5 +1,5 @@
 {
-  "version": "0.2.0",
+  "version": "0.3.0",
```

Comment:
If we can solve the lerna dependencies stuff in here, I'm happy to inherit the success into my Icon PR 😅

Reply:
I havent been tracking the lerna issues. What are they? I there an issue for them?

Reply:
No issue logged - you can check out the problem I'm having here: https://github.com/okta/odyssey/pull/573 

I'm assuming we may see something similar when you add a dependency to the vuepress theme.

I'd recommend adding it to the package via `lerna add`, so that we're relying on lerna for all of our project management.